apps/experiments/views/experiment.py

```diff
@@ -217,11 +218,11 @@ def save(self, commit=True):
 
 def _validate_prompt_variables(form_data):
     required_variables = set(PromptTemplate.from_template(form_data.get("prompt_text")).input_variables)
-    available_variables = set()
+    available_variables = set(["participant_details"])
```

Comment:
@bderenzi since we're about to introduce this as a variable that can be included in a prompt, do you want to suggest any alternative names? (we can add an alias later if necessary but easier to change it now).

During our call you mentioned that the term you'd been using was `user profile`. Should we go with that?

Reply:
No strong feelings. Both `{participant_details}` and `{user_profile}` are 4 tokens each in the OpenAI world. In my brain, `user_profile` makes a little more sense. Or maybe even `participant_profile`. I think the world is pretty used to the idea of a profile being properties about an individual. 

I suspect in a lot of cases, we'll want to pull out specific details to use them in the prompt.

For fun, I asked ChatGPT a handful of times. It seemed to gravitate to `user_info`, `user_profile`, `user_context`, and `user_data`  as being simple, concise, and versatile options. Take that as you will--the robots voted strongly for starting with `user`. Of these options, `user_info` and `user_data` stand out to me as being pretty clear. Happy to defer to you all (though can pick if you want a tie breaker)

Reply:
Thanks for going through all the trouble @bderenzi :smile:  `user_data` sounds good. I'll make an update

Reply:
What's the confusion? You mean that `user` might be the prompt author instead of the chat participant? I don't think people would be confused. I also think we'll have to manually specify the properties that we want to track in `user_data` somewhere as well. 

Also, I'm happy with `participant_data` if that seems more clear.

Reply:
Is now a reasonable time to add the alias and use both `user_data` and `participant_data`?

Reply:
I think let's keep the alias for a separate PR. I want to make an update some time to show the user the available variables with a mini description. I'll ship the alias changes with that if we still want one. Questions on the alias: Should this be user defined, or do we want to have this statically defined?

Reply:
I was thinking statically defined.

Reply:
Oh ok then it should be a quick one